yarl/__init__.py

```diff
@@ -192,6 +192,23 @@ def __init__(self, val='', *, encoded=False, strict=False):
         self._val = val
         self._cache = {}
 
+    @classmethod
+    def build(cls, scheme='', user='', password='', host='', port=None, path='',
+              query_string='', fragment='', *, strict=False, encoded=False):
+
+        assert scheme and host, "You must specify scheme and host"
+
+        return cls(
+            urlunsplit(
```

Comment:
`SplitResult` has been removed.
`urlunsplit` implementation is so simple and clear (just string concatenation and condition checks)